src/SqlPersistence/Outbox/OutboxPersister.cs

```diff
@@ -15,11 +15,13 @@
 class OutboxPersister : IOutboxStorage
 {
     Func<DbConnection> connectionBuilder;
+    int cleanupBatchCount;
     OutboxCommands outboxCommands;
 
-    public OutboxPersister(Func<DbConnection> connectionBuilder, string tablePrefix, string schema, SqlVariant sqlVariant)
+    public OutboxPersister(Func<DbConnection> connectionBuilder, string tablePrefix, string schema, SqlVariant sqlVariant, int cleanupBatchCount = 10000)
```

Comment:
> cleanupBatchCount 

IMO cleanupBatchSize is a better name, as count might indicate how many batches we want to divide that into